src/test/java/io/strimzi/kafka/bridge/http/StaticTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.http;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+public class StaticTest {
+
+    public String getVersionFromFile(String releaseFile) throws Exception {
+
+        String versionFromFile;
+
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(releaseFile));
+            versionFromFile = bufferedReader.readLine();
+            bufferedReader.close();
+        } catch (FileNotFoundException e) {
+            throw new Exception("File not found : " + releaseFile);
+        } catch (IOException e) {
+            throw new Exception("Unable to open file : " + releaseFile);
+        }
+
+        if ((versionFromFile == null) || (versionFromFile.isEmpty())) {
+            throw new Exception("Unable to get Version from file : " + releaseFile);
+        }
+        return versionFromFile;
+    }
+
+    @Test
+    void bridgeVersionDisplayedInStartupTest() throws Exception {
+
+        String kBVersion = getVersionFromFile("release.version");
+        assertThat(kBVersion, is(not("Unable to get version")));
+        Boolean versionFound = false;
```

Comment:
I think this variable is not used anywhere anymore.

Reply:
ok, removed